test/conformance/walk.go

```diff
@@ -174,6 +174,7 @@ func (v *visitor) comment(x *ast.BasicLit) string {
 			//if we fail to open the file to compare the content we just assume the
 			//proximity of the comment and apply it.
 			myf, err := os.Open(v.FileSet.File(x.Pos()).Name())
+			defer myf.Close()
```

Comment:
Should this `defer myf.Close()` be in the `err == nil` block? Otherwise, we may try to close `myf` even though it wasn't opened successfully?